platform/src/pages/analytics/tabs/OverView.jsx

```diff
@@ -1,15 +1,85 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import ChartContainer from '@/components/Charts/ChartContainer';
+import AQNumberCard from '@/components/AQNumberCard';
+import BorderlessContentBox from '@/components/Layout/borderless_content_box';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRecentMeasurementsData } from '@/lib/store/services/deviceRegistry/RecentMeasurementsSlice';
 
 const OverView = () => {
+  // events hook
+  const dispatch = useDispatch();
+  const recentLocationMeasurements = useSelector((state) => state.recentMeasurements.measurements);
+  const chartDataRange = useSelector((state) => state.chart.chartDataRange);
+  const userDefaults = useSelector((state) => state.userDefaults.defaults);
```

Comment:
Hi @Codebmk , since we are moving towards using Preferences, please consider using `Preferences` endpoints for this. For context, `Preferences` was implemented last week and it is fully ready to be utilised. 

**Background to this change from Defaults to Preferences:** 

- It was discovered that Defaults had so many issues that could not be fixed so we just decided to create a completely new set of endpoints for Analytics Redesign. 
- We might also to discontinue the usage of Defaults in the Old Analytics (Netmanager) and completely switch to Preferences all throughout our Systems.

Reply:
list user preferences api says not found ie `{{baseUrl}}/api/v2/users/preferences/USERID`
![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/9e7c1cb5-bb69-4809-a624-8fb7242b87a4)